lib/fakerbot/renderer.rb

```diff
@@ -73,5 +64,20 @@ def gt_screen_height?(data_tree)
     def verbose?
       options[:verbose]
     end
+
+    def verbose_output(method, const, arr)
+      fake, message = faker_method(method, const)
+      arr << crayon.dim.white("=> #{fake.to_s}") << crayon.dim.magenta.bold("#{message}")
+    end
+
+    def faker_method(method, const)
+      [const.public_send(method), deprecate(method, const)]
+    rescue ArgumentError
+      ['N/A', '']
+    end
+
+    def deprecate(method, const)
+      const.respond_to?("_deprecated_#{method.to_s}".to_sym) ? ' ( WILL BE DEPRECATED )' : ''
```

Comment:
Can we use direct symbol interpolation here? `:"_deprecated_#{method}"`?

Reply:
Thank you. 😄